esmvaltool/recipes/recipe_autoassess_radiation_rms_Amon_obs.yml

```diff
@@ -1,10 +1,18 @@
+# ESMValTool
+# recipe_autoassess_radiation_rms_Amon_obs.yml
 ---
 documentation:
-  description: Diagnostics of spatial RMS errors of radiation and clouds.
+  description: |
+    Diagnostics of spatial RMS errors of radiation and clouds.
+
   authors:
     - tsus_yo
     - read_si
```

Comment:
same here for Yoko and Simon; I'd add Alistair Sellar @alistairsellar to replace them